CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Rojo Changelog
 
+## [7.4.3] - 31 July, 2024
```

Comment:
You should put this in the `## Unreleased Changes` section, you are not making a new release.